ngraph/python/CMakeLists.txt

```diff
@@ -73,11 +77,35 @@ file(GLOB_RECURSE SOURCES src/pyngraph/*.cpp)
 pybind11_add_module(_${PROJECT_NAME} MODULE ${SOURCES})
 
 target_include_directories(_${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
-target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::ngraph)
+target_include_directories(_${PROJECT_NAME} PRIVATE "${FRONTEND_INCLUDE_PATH}")
+
+if (NGRAPH_ONNX_IMPORT_ENABLE)
+    target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::onnx_importer)
+endif()
+
+target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::ngraph ngraph::frontend_manager)
+
 if (TARGET ngraph::onnx_importer)
     add_dependencies(_${PROJECT_NAME} ngraph::onnx_importer)
 endif()
 
+if(NGRAPH_UNIT_TEST_ENABLE)
+    set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY_OLD})
+    set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_ARCHIVE_OUTPUT_DIRECTORY_OLD})
+    set(CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY ${CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY_OLD})
+    set(CMAKE_PDB_OUTPUT_DIRECTORY ${CMAKE_PDB_OUTPUT_DIRECTORY_OLD})
+    add_subdirectory(tests/mock/mock_py_ngraph_frontend)
```

Comment:
Can we avoid changing of `CMAKE_LIBRARY_OUTPUT_DIRECTORY`?
I think that something like `set_target_properties(${PYTHON_TARGET} PROPERTIES LIBRARY_OUTPUT_DIRECTORY  ${PYTHON_BRIDGE_OUTPUT_DIRECTORY})` should work as well.

Reply:
I see that these 4 items (library, archive, pdb, pdb_compile) is used in lot of places in a project and thought that it is correct way to override this. I tried your suggestion, seems it works for Linux, let's see other platforms from CI pre-commit